WcaOnRails/app/models/wfc_dues_redirect.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class WfcDuesRedirect < ApplicationRecord
+  belongs_to :redirect_to, class_name: "WfcXeroUser", foreign_key: "redirect_to_id"
+
+  enum :redirect_type, {
+    country: "country",
+    organizer: "organizer",
+  }
+
+  def redirect_from_country
+    if redirect_from_country_id.nil?
+      return nil
+    end
+    Country.find_by_id(redirect_from_country_id)
+  end
+
+  def redirect_from_organizer
+    if redirect_from_organizer_id.nil?
+      return nil
+    end
+    User.find(redirect_from_organizer_id)
+  end
```

Comment:
This can be completely skipped based on my other comment in this file.

Reply:
Done